crates/viewer/re_view_graph/src/view.rs

```diff
@@ -105,9 +105,11 @@ impl ViewClass for GraphView {
             .get(&NodeVisualizer::identifier())
         {
             ViewSpawnHeuristics::new(maybe_visualizable.iter().cloned().filter_map(|entity| {
-                suggested_filter
-                    .matches(&entity)
-                    .then_some(RecommendedView::new_single_entity(entity))
+                if !suggested_filter.matches(&entity) {
+                    Some(RecommendedView::new_single_entity(entity))
+                } else {
+                    None
+                }
```

Comment:
let's at least avoid double negatives

```suggestion
                if suggested_filter.matches(&entity) {
                    None
                } else {
                    Some(RecommendedView::new_single_entity(entity))
                }
```